src/server/api/routes/bookstore/index.js

```diff
@@ -0,0 +1,55 @@
+const { Router } = require('express');
+const { Client } = require('@notionhq/client');
+
+const { IS_TESTNET, NOTION_API_KEY } = require('../../../config/config');
+
+const { handleRestfulError } = require('../../middleware/error');
+
+const router = Router();
+
+const notion = new Client({ auth: NOTION_API_KEY });
+
+router.get('/bookstore/listing', async (req, res, next) => {
+  try {
+    const response = await notion.databases.query({
+      database_id: '755267a797e146869f74bf13be788ca5',
```

Comment:
Should this be in config file too?

Reply:
Moved the id to the config file and separated database for testnet and production